vllm_hpu_extension/ops.py

```diff
@@ -209,6 +223,15 @@ def prompt_attention(
         if query_heads != kv_heads:
             attn_weights = attn_weights.flatten(1, 2)
     else:
+        VLLM_FUSEDSDPA_REPEAT_KV_CACHE = os.environ.get('VLLM_FUSEDSDPA_REPEAT_KV_CACHE','0') == '1'
+        VLLM_FUSEDSDPA_REPEAT_KV_CACHE_SPLIT_GRAPHS = os.environ.get('VLLM_FUSEDSDPA_REPEAT_KV_CACHE_SPLIT_GRAPHS','0') == '1'
```

Comment:
Can you rename the variables? It is already known at this stage that it is "VLLM" flag and that it is FSDPA code. 
So it will be better to simply rename it to something like:
`do_repeat_kv_cache` and `do_split_repeat_kv_cache_graphs`

Reply:
done